### PR TITLE
Spelling/grammar/wording/etc fixes

### PR DESCRIPTION
**Pull request by Wizardcrying**
The ultimate improvement to the shift-clicking experience ***and more!***

I ended up including the entire `vending.dm` file in the first commit but I started with some other changes. 
Feel free to comment on anything I've changed or any changes I should make.

:cl: 
 * bugfix: Fixed many spelling/grammar/standardization/et cetera issues, especially with examine text and Syndicate uplink descriptions.

### CODE REVIEW

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 /obj/item/clothing/under/rank/security/sneaksuit
 	name = "sneaking suit"
-	desc = "It's made of a slightly sturdier material developed by the Soviet Union centuries ago, allows for robust protection."
+	desc = "It's made of a strong material developed by the Soviet Union centuries ago, allows for robust protection."
```
Comment (Intigracy) [inline code/modules/clothing/under/jobs/security.dm]: no comma, turn it into `which provides robust protection`

File: code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -9,7 +9,7 @@
  * Security
  */
 /obj/item/clothing/under/rank/warden
-	desc = "It's made of a slightly sturdier material than standard jumpsuits, to allow for more robust protection. It has the word \"Warden\" written on the shoulders."
+	desc = "A jumpsuit made of strong material, providing better protection. It has the word \"WARDEN\" written on the shoulders."
```
Comment (Intigracy) [inline code/modules/clothing/under/jobs/security.dm]: all the ones that say `providing better protection` should probably say `providing it better protection` or `providing robust protection`

File: code/modules/recycling/conveyor2.dm [hunk 3]
```diff
@@ -9,7 +9,7 @@
 	icon_state = "conveyor0"
 	name = "conveyor belt"
 	desc = "A conveyor belt.\
-	<br><span class='info'>It looks like it can be pried into a different direction using a crowbar but cannot be moved without welding it apart.</span>"
+	<br><span class='info'>It can be pried into a different direction using a crowbar but cannot be moved without welding it apart.</span>"
```
Comment (Intigracy) [inline code/modules/recycling/conveyor2.dm]: comma before but

**Comment by Wizardcrying** [comment]
what the fuck

**Review by PJB3005 — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 4]
```diff
@@ -3437,7 +3437,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/sliceable/buchedenoel
 	name = "\improper Buche de Noel"
-	desc = "Merry Christmas!"
+	desc = "Merry Christmas"
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: But that one was fine?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 5]
```diff
@@ -3437,7 +3437,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/sliceable/buchedenoel
 	name = "\improper Buche de Noel"
-	desc = "Merry Christmas!"
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: That was fine though.

**Comment by Probe1** [comment]
I believe in you @Wizardcrying

**Comment by Wizardcrying** [comment]
thanks @Probe1

**Comment by Wizardcrying** [comment]
I think that's it for this round folks, waiting for review, any further needed changes and merge
**BUT NOT BEFORE I DO `uplink_item.dm`!**

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 6]
```diff
@@ -219,7 +219,7 @@
 
 /obj/item/weapon/storage/box/wind
 	name = "wind grenades"
-	desc = "A box containing 3 wind grenades"
+	desc = "A box containing wind grenades."
```
Comment (Probe1) [inline code/game/objects/items/weapons/storage/boxes.dm:222]: I'll take "Shit I didn't even know was in the code" for 500, Alex

File: code/modules/detectivework/evidence.dm [hunk 7]
```diff
@@ -84,7 +84,7 @@ obj/item/weapon/evidencebag/attackby(obj/item/weapon/W as obj, mob/living/user a
 
 /obj/item/weapon/storage/box/evidence
 	name = "evidence bag box"
-	desc = "A box claiming to contain evidence bags."
+	desc = "A box containing evidence bags."
 	New()
```
Comment (Probe1) [inline code/modules/detectivework/evidence.dm:88]: Mental note to clean this shit code up.

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 5 above)
Comment (Probe1) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: If he's gonna clean up the bullshit let em clean up the bullshit.

**Comment by Probe1** [comment]
>I'm done
 >Just kidding editing my post saying I'm done to say I'm doing more
TRUST ISSUES.

**Review by Probe1 — APPROVED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 6 above)
Comment (Wizardcrying) [reply code/game/objects/items/weapons/storage/boxes.dm:222]: I think I discovered quite a few things on this mystical journey.
Thanks `ctrl+shift+f`!

File: code/modules/clothing/under/chameleon.dm [hunk 8]
```diff
@@ -58,7 +58,7 @@
 			return
 
 		var/obj/item/clothing/under/A
-		A = input("Select colour to change to.", "BOOYEA", A) in clothing_choices
```
Comment (PJB3005) [inline code/modules/clothing/under/chameleon.dm]: How to give celt a heart attack.

Comment (Wizardcrying) [reply code/modules/clothing/under/chameleon.dm]: hehehe

File: maps/ministation/uplink_item.dm [hunk 9]
```diff
@@ -20,7 +20,7 @@
 	..()
 	item = null // Disabled
 
-/datum/uplink_item/stealthy_tools/syndigolashes/New()
+/datum/uplink_item/stealthy_tools/syndigaloshes/New()
```
Comment (Probe1) [inline maps/ministation/uplink_item.dm:23]: ```  /datum/uplink_item/nukeprice/syndigolashes ```

see above line in same file and edit.  also careful now you're changing paths not desc.

**Review by Probe1 — CHANGES_REQUESTED** [review]
do the needful for syndigolashes please.

File: maps/ministation/uplink_item.dm (hunk 9 above)
Comment (Wizardcrying) [inline maps/ministation/uplink_item.dm]: This isn't defining the nuke price?

Comment (Wizardcrying) [reply maps/ministation/uplink_item.dm:23]: It should already be edited.

**Review by Probe1 — APPROVED** [review]

File: maps/ministation/uplink_item.dm (hunk 9 above)
Comment (Probe1) [reply maps/ministation/uplink_item.dm:23]: I thought this was the actual file.  I'm an ijiot

Comment (Intigracy) [reply maps/ministation/uplink_item.dm:23]: ensure this is in none of the map files (check vaults too)

Comment (Shadowmech88) [reply maps/ministation/uplink_item.dm:23]: If it was in map files/vaults, wouldn't it cause a compile failure now that their path no longer exists?

File: code/modules/clothing/under/chameleon.dm [hunk 10]
```diff
@@ -54,11 +54,11 @@
 		set src in usr
 
 		if(icon_state == "psyche")
-			to_chat(usr, "<span class='warning'>Your suit is malfunctioning</span>")
+			to_chat(usr, "<span class='warning'>Your suit is malfunctioning.</span>")
 			return
 
 		var/obj/item/clothing/under/A
-		A = input("Select Colour to change it to", "BOOYEA", A) in clothing_choices
+		A = input("Select color to change to.", "BOOYEA", A) in clothing_choices
```
Comment (ghost) [inline code/modules/clothing/under/chameleon.dm]: this is non atomic

**Review by ghost — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/chameleon.dm (hunk 10 above)
Comment (Wizardcrying) [reply code/modules/clothing/under/chameleon.dm]: no

File: maps/ministation/uplink_item.dm (hunk 9 above)
Comment (Wizardcrying) [inline maps/ministation/uplink_item.dm]: This IS the map counterpart. Maps would't use datums unless they use it like this. `code/datum/uplink_item.dm` is what you're looking for @Intigracy

Comment (Wizardcrying) [reply maps/ministation/uplink_item.dm:23]: This IS the map counterpart. Maps wouldn't use datums unless they use it like this. `code/datum/uplink_item.dm` is what you're looking for @Intigracy

**Pull request merged**